lib/apiservers/engine/backends/image.go

```diff
@@ -112,6 +112,7 @@ func (i *Image) PullImage(ctx context.Context, ref reference.Named, metaHeaders
 	defer trace.End(trace.Begin("PullImage"))
 
 	log.Debugf("PullImage: ref = %+v, metaheaders = %+v\n", ref, metaHeaders)
+	log.Printf("* PullImage - authConfig = %#v", authConfig)
```

Comment:
This should be Debugf, and only if authConfig doesn't contain sensitive data such as username/password/auth token